batcher/aligned-batcher/src/lib.rs

```diff
@@ -1786,7 +1795,12 @@ impl Batcher {
             ETHEREUM_CALL_MAX_RETRY_DELAY,
         )
         .await
-        .map_err(|e| BatcherError::BatchUploadError(e.to_string()))
+        .map_err(|e| BatcherError::BatchUploadError(e.to_string()))?;
+
+        self.metrics
+            .s3_latency
+            .set(start.elapsed().as_micros() as i64);
```

Comment:
Did we check if our S3 client already provides a metrics exporter?

Reply:
Created #1583 to review this!